### PR TITLE
LPS-152122 SamlProviderConfiguration via REST

### PR DESCRIPTION
**Pull request by stian-sigvartsen**
Relates to subtask: https://issues.liferay.com/browse/LPS-152126

### CODE REVIEW

**Comment by stian-sigvartsen** [comment]
ci:stop

File: modules/dxp/apps/saml/saml-admin-rest-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+                    example: EntityID too long (Max 1024 characters)
+                    readOnly: true
+                    type: string
+                type:
+                    description:
+                        Type of problem
+                    readOnly: true
+                    type: string
+            required:
+                - message
```
Comment (oliverkecskemety) [inline modules/dxp/apps/saml/saml-admin-rest-impl/rest-openapi.yaml:28]: This property does not exist

**Review by oliverkecskemety — CHANGES_REQUESTED** [review]
As we agreed the `Error` schema is unnecessary in the `rest-openapi.yaml` file. Don't forget to put a period at the end of the descriptions. Otherwise everything looks good to me. :)

**Comment by stian-sigvartsen** [comment]
Thanks for reviewing @oliverkecskemety ! :)

**Comment by stian-sigvartsen** [comment]
See liferay-appsec/liferay-portal/pull/777

**Pull request closed without merging**